restructured-document/requirements.tex

```diff
@@ -0,0 +1,34 @@
+\section{Requirements}
+
+\subsection{Mandatory}
+\begin{itemize}
+\item JA must assign user jobs to work machines.
+ \item JA must provide a user client on which a user can make job requests.
+\item JA must be accessible through a CLI.
+\item Users can specify a priority for a job.
+\item Users can specify the minimum hardware requirements for a job: RAM, CPU thread count, GPU memory/model/speed.
+\item Users can specify an operating system on which the job should be run.
+\item The system must store all the data of queued, running and fulfilled jobs.
```

Comment:
I suggest:

JA must store the metadata of all queued, running, and past jobs in a database.